docker-compose.yml

```diff
@@ -94,21 +96,21 @@ services:
     environment:
       POSTGRES_PASSWORD: postgres
     ports:
-      - "5432:5432"
+      - '5432:5432'
     volumes:
       - db_volume:/var/lib/postgresql
 
   redis:
     image: redis
     ports:
-      - "6379:6379"
+      - '6379:6379'
```

Comment:
🤔 Are these quote styles going to flip-flop between the last person editing the file? Or was this programmatically formatted?